corehq/blobs/management/commands/import_blob_zip.py

```diff
@@ -1,11 +1,22 @@
 import io
 import zipfile
+from inspect import cleandoc
```

Comment:
`cleandoc` is handy, I need to remember this. Who would have known it lives in `inspect` though?

Reply:
Yeah. (As you can tell, for inspecting docstrings, so they look consistent regardless of how people write them.) I discovered [`textwrap.dedent()`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) from reviewing Ethan's code, but `cleandoc()` also cleans up newlines.